brave/brave-core
# Transition Brave Ads prefs from the service layer to bat-native-ads

Resolves https://github.com/brave/brave-browser/issues/9999

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm ads per hour setting is persisted across browser restarts
- Confirm idle threshold for ad delivery is persisted across browser restarts
- Confirm ad conversions are working as expected
- Confirm subdivision targeting is working as expected

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
